inbm/dispatcher-agent/tests/unit/test_dispatcher_class.py

```diff
@@ -96,12 +96,14 @@ def test_run_one_immediate_scheduled_manifest(mock_disp_obj, method_counter, moc
     # Assert that the do_install method is called once
     assert method_counter.call_count == 1
 
-@patch('dispatcher.schedule.sqlite_manager.SqliteManager.create_task')
-def test_run_several_immediate_scheduled_manifest(mock_create_task, mock_disp_obj, method_counter, mocker):
+
+@patch('dispatcher.schedule.sqlite_manager.sqlite3.connect')
```

Comment:
<h3>$\color{#FFA500}Improvements: Code Maintainability$</h3>

Remove unused mock patch `mock_conn` to clean up the test code.

Reply:
Test fails if mock_conn is removed